src/main/java/uk/gov/companieshouse/api/accounts/validation/ApprovalValidator.java

```diff
@@ -31,7 +49,25 @@ public Errors validateApproval(Approval approval, HttpServletRequest request) {
                 ErrorType.VALIDATION.getType()));
         }
 
+        ResponseObject<Director> directorsReportResponseObject = directorService.findAll(transaction, companyAccountId, request);
+
+        if(directorsReportResponseObject != null) {
```

Comment:
For future ref, you don't need this null check either; directorService#findAll will never return null

Reply:
Removed the unwanted if and sorted the tests out